browser/src/slideshow/engine/SlideShowNavigator.ts

```diff
@@ -244,8 +262,59 @@ class SlideShowNavigator {
 	}
 
 	private clickHandler(aEvent: MouseEvent) {
-		if (aEvent.button === 0) this.dispatchEffect();
-		else if (aEvent.button === 2) this.switchSlide(-1, false);
+		if (aEvent.button === 0) {
```

Comment:
apart of click we should also handle "hover" so we can use "hand" mouse pointer to indicate interactive element for the user